components/footer.js

```diff
@@ -20,20 +20,18 @@ export default function Footer() {
             alt="imageIcon"
             height={50}
             width={50}
-            objectFit="contain"
           />
         </Link>
       </NavElement>
 
-      <NavElement active={pathname.includes("/favorites")}>
-        <Link href="/favorites">
+      <NavElement active={pathname.includes("/favoritespage")}>
+        <Link href="/favoritespage">
           <Image
```

Comment:
Only a minor thing: there is no need to have the word "page" in an url, because it always represents a page. I would therefore stick to "/favorites".

Reply:
Done.